build/NuGetAdditionalFiles/Microsoft.NETCore.Compilers.props

```diff
@@ -29,7 +31,9 @@
   <!-- If packaged on Windows, the RunCsc and RunVbc scripts may not be marked
        executable. If we're not running on Windows, mark them as such, just in case -->
   <Target Name="MakeCompilerScriptsExecutable"
-          Condition="'$(BuildingProject)' == 'true' AND '$(OS)' != 'Windows_NT'"
+          Condition="'$(BuildingProject)' == 'true' AND
+                     '$(OS)' != 'Windows_NT' AND
+                     '$(CscToolPath)' != ''"
```

Comment:
Don't agree with the `CscToolPath` condition here. Consider the case where only `CscToolPath` was set but not `VbcToolPath`. In that case we still need to make the scripts executable.

Reply:
Yeah good point.